css/style.css

```diff
@@ -232,10 +248,18 @@ hr {
 
 .about p {
     font-size: 23px;
+    color: whitesmoke;
```

Comment:
Instead of named color use the hexcode so that the range of browser support is more.